Mathlib/Tactic/ToAdditive.lean

```diff
@@ -731,6 +731,7 @@ def nameDict : String → List String
   | "prehaar"     => ["add", "Prehaar"]
   | "unit"        => ["add", "Unit"]
   | "units"       => ["add", "Units"]
+  | "prods"       => ["sums"]  -- convert `[uU]niqueProds` to `[uU]niqueSums`
```

Comment:
This feels a bit dangerous to me. Can you add it to `fixAbbreviation` instead and match the `unique` part too?

Reply:
I had tried that first, but I could not get it to work.  I think that the splitting of words happens before the `fixAbbreviation` has a chance to process the string... *I think*.*

Reply:
Can't you follow the `"Is" :: "Scalar" :: "Tower"` pattern that we use there?